### PR TITLE
Bug 1741675, added support note about upgrade

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1741675

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@rjhowe @lihongan PTAL. Thank you!

**Comment by lihongan** [comment]
LGTM. Thanks.

File: install_config/router/customized_haproxy_router.adoc [hunk 1]
```diff
@@ -49,6 +49,15 @@ updated, the image needs to be rebuilt and pushed.
 
 In either case, the router pod starts with the template file.
 
+[IMPORTANT]
+====
+It is not guaranteed that router template customizations will work when
```
Comment (kalexand-rh) [inline install_config/router/customized_haproxy_router.adoc:54]: Do you mean "will still work after you upgrade {product-title}?"

File: install_config/router/customized_haproxy_router.adoc [hunk 2]
```diff
@@ -49,6 +49,15 @@ updated, the image needs to be rebuilt and pushed.
...
+upgrading {product-title}.
+
+Also, router template customizations must be applied to the template version of
```
Comment (kalexand-rh) [inline install_config/router/customized_haproxy_router.adoc:57]: I am not sure what you mean here, but I think it goes after line 40.

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions!

**Pull request merged**